packages/google_sign_in/example/lib/main.dart

```diff
@@ -0,0 +1,173 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// ignore_for_file: public_member_api_docs
+
+import 'dart:async';
+import 'dart:convert' show json;
+
+import 'package:flutter/material.dart';
+import 'package:google_sign_in/google_sign_in.dart';
+import 'package:google_sign_in_tizen/google_sign_in_tizen.dart';
+import 'package:http/http.dart' as http;
+
+// `credentials.dart` contains the actual "client id" and "client secret".
+// `fake_credentials.dart` is a dummy file to pass CI.
+import 'fake_credentials.dart' as credentials;
+// import 'credentials.dart' as credentials;
```

Comment:
I'm not entirely sure on what you meant, are changes in the next commit OK?

Reply:
Yes, exactly. But did you notice that you included your own credentials in the commit?

Reply:
It was intentional so that the example app can be tested without having to create another credential, I'm not planning to erase it anytime soon (If I do, contributors will need to put their own credentials as instructed in the comments). Plus, `credentials.dart` file will not be published.